src/App.tsx

```diff
@@ -1,24 +1,182 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useEffect,
+  useState,
+  ChangeEvent,
+  FormEvent,
+} from 'react';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { ErrorPanel } from './components/ErrorPanel';
+
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+
+import { getTodos, addTodo, removeTodo } from './api/todos';
+
+const USER_ID = 6160;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
```

Comment:
I guess you can create a separate enum for the error type and make this state typeof this enum. And change it in the `catch`. Will make more sense this way

Reply:
thank you for your review, working on them